phone-scripts/02-install-packages.sh

```diff
@@ -29,14 +29,11 @@ infecho "THIS WILL FAIL, DON'T WORRY ITS PROBABLY OK"
 dnf -y remove kernel || rpm -e --noscripts kernel-core
 dnf -y install linux-firmware
 
-infecho "Installing kernel..."
-dnf -y --disablerepo="*" --enablerepo="copr:copr.fedorainfracloud.org:njha:mobile" install megi-kernel
-
 infecho "Installing recommended packages..."
-dnf -y install feedbackd phoc phosh squeekboard gnome-shell ModemManager rtl8723cs-firmware \
+dnf -y install megi-kernel feedbackd phoc phosh squeekboard gnome-shell ModemManager rtl8723cs-firmware \
     f2fs-tools chatty calls carbons purple-mm-sms pinephone-helpers evolution-data-server \
-    f32-backgrounds-gnome kgx epiphany gnome-contacts evolution cheese NetworkManager-wwan \
```

Comment:
Grabbing f33 backgrounds and removing cheese, as it does nothing anyway on PinePhone/PineTab